test/test_experiment_entrypoint.py

```diff
@@ -114,7 +115,7 @@ def d(key):
         for i in range(len(timestamps)):
             prev = 0.0 if i == 0 else timestamps[i - 1]
             cur = timestamps[i]
-            self.assertGreater(cur, prev)
+            self.assertGreaterEqual(cur, prev)
```

Comment:
Did you actually run into an issue here? I used strict inequality on purpose to reject the case where e.g. all timestamps are just set to the same value by accident, since the granularity of time.monotonic should be considerably better than microseconds.

Reply:
I did, but it's possible that it was unrelated and I didn't dig into it properly (e.g. could have been an `artiq` version conflict?). This is a bit sloppy anyway. I'll revert this change and open an issue if I hit it again.